File: Day_1/Q1.java
import java.util.*;
class Q1{
    //Calculating And Printing The Sum
    static int sum_N(int n){
        return (n*(n+1)/2);
    }
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        if(n <= 0) System.out.println("Invalid Input"); //Checking Valid Input
        else System.out.println("The Sum Of First "+n+" Natural Numbers Is " + sum_N(n));
        sc.close();
    }
}

File: Day_1/Q1.cpp
#include <iostream>
using namespace std;
// Calculating The Sum
int sum_N(int n)
{
    return (n*(n+1))/2;
}
int main()
{
    int n;
    cin >> n;
    //If Input Is Not Valid 
    if(n <= 0){
        cout << "Invalid Input";
        return 0;
    }
    //Printing The Result
    cout << "The Sum Of First " << n << " Natural Numbers Is " << sum_N(n);
    return 0;
}
